odl/contrib/fom/supervised.py

```diff
@@ -72,10 +78,15 @@ def mean_squared_error(data, ground_truth, mask=None, normalized=False):
     else:
         fom /= l2norm_squared(data.space.one())
 
+    # The mean squared error is already in the desired order.
+    if force_lower_is_better:
+        pass
```

Comment:
Just a comment suffices, the code really does nothing.

Reply:
Our motivation for adding this was that any compiler would complain about the unused variable, and so did the syntax checker of my Emacs environment. If this warning is not a problem for ODL, we can of course leave this out.

Reply:
Yes, pylint would probably complain, too (maybe that's what Emacs uses). But in general I'm against adding code just to silence style checkers.